### ISSUE TITLE
offline

### ISSUE THREAD

**Issue by hugurp** [hugurp/note#22]
once this aplication has been opened once in the browser, it would be great to have everything work offline. 

So if there is no internet connection, the app still loads, and it is possible to read notes, or at least to take some new one.

New note should be merged to the existing notes when internet is back

**Comment by oskarrough** [comment]
I believe remotestorage has some offline support built-in. So maybe, all we need to do is add a service worker? Come to think of it, create-react-app comes with one https://github.com/hugurp/note/blob/master/src/index.js#L17-L20

**Comment by hugurp** [comment]
True, I need to investigate more on how to setup the asset caching, because you're right remote storage supports it. If the network is cut, the app still works, but only if the assets are already loaded.
I'm guessing the service worker is here to help.

**Comment by hugurp** [comment]
Also, because it is a web app, it will be really nice on mobile

**Comment by oskarrough** [comment]
Can consider using the workbox-cli to get a starting sw.js https://developers.google.com/web/tools/workbox/guides/generate-service-worker/cli
